### PR TITLE
Fit screen iphone se

### PR DESCRIPTION
**Pull request by jankrzempek**
https://trello.com/c/cu6jE4eo/382-scale-screen-so-it-fits-iphonese

To test, please use Ipod touch 7 simulator

### CODE REVIEW

File: AirCasting/SessionViews/ABMeasurementsView.swift [hunk 1]
```diff
@@ -62,20 +62,17 @@ struct _ABMeasurementsView: View {
...
+                                                      threshold: threshold,
+                                                      selectedStream: $selectedStream,
+                                                      isCollapsed: $isCollapsed,
+                                                      measurementPresentationStyle: measurementPresentationStyle,
+                                                      isDormant: session.isDormant)
                             }
+                            Spacer()
                         }
-                        .padding(.horizontal, 8)
-                        .frame(maxWidth: .infinity, alignment: .leading)
```
Comment (smialko) [inline AirCasting/SessionViews/ABMeasurementsView.swift:78]: Should AB measurements be on the left side of the session card? I think not but tbh I'm not sure. I think that removing the frame and padding moved them to the left. Also, they seem more condensed than previously. 
IMG_F07B0AE0675D-1

Comment (smialko) [reply AirCasting/SessionViews/ABMeasurementsView.swift:78]: Uh, now I see the `Spacer()` - it's its fault the measurements are on the left side.

**Review by smialko — CHANGES_REQUESTED** [review]
Great job! Just please fix the ABMeasurementsView :)

File: AirCasting/SessionViews/ABMeasurementsView.swift (hunk 1 above)
Comment (smialko) [reply AirCasting/SessionViews/ABMeasurementsView.swift:78]: For comparison I'm attaching a photo of how it looked before: 

image

**Review by pawelgil — APPROVED** [review]
Note: I only review the code assuming it's working as expected

File: AirCasting/SessionViews/ABMeasurementsView.swift [hunk 2]
```diff
@@ -111,16 +109,17 @@ struct _ABMeasurementsView: View {
...
             Group {
+                streamsToShow.count != 1 ? Spacer() : nil
                 ForEach(streamsToShow, id : \.self) { stream in
                     SingleMeasurementView(stream: stream,
                                           threshold: nil,
                                           selectedStream: $selectedStream,
                                           isCollapsed: $isCollapsed,
                                           measurementPresentationStyle: .hideValues,
                                           isDormant: session.isDormant)
+                    Spacer()
```
Comment (smialko) [inline AirCasting/SessionViews/ABMeasurementsView.swift:120]: I don't think using a Spacer() here is a good idea, right now the AB parameters are squeezed in the middle and there's too much space on the sides. Maybe try to use horizontal `.padding()` instead? :)

Comment (smialko) [reply AirCasting/SessionViews/ABMeasurementsView.swift:120]: Here is just an example, don't hesitate to do it another way:

```
private var streamNames: some View {
        return HStack {
            Group {
                ForEach(streamsToShow, id : \.self) { stream in
                    SingleMeasurementView(stream: stream,
                                          threshold: nil,
                                          selectedStream: $selectedStream,
                                          isCollapsed: $isCollapsed,
                                          measurementPresentationStyle: .hideValues,
                                          isDormant: session.isDormant)
                    streamsToShow.count == 1 ? Spacer() : nil
                }
            }
            .padding(.horizontal)
        }
    }
```

**Review by smialko — COMMENTED** [review]
it's better now! 😄

**Pull request merged**